samples/excel/85-preview-apis/comment.yaml

```diff
@@ -0,0 +1,167 @@
+order: 1
+id: excel-comment
+name: Comments
+description: 'Adds, edits, and removes comments and comment replies.'
+host: EXCEL
+api_set:
+    ExcelApi: 1.9
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup)); $("#addCommentToSelectedCell").click(() => tryCatch(addCommentToSelectedCell)); $("#addCommentToCell").click(() => tryCatch(addCommentToCell)); $("#addCommentReply").click(() => tryCatch(addCommentReply)); $("#getCommentMetadata").click(() => tryCatch(getCommentMetadata)); $("#editComment").click(() => tryCatch(editComment)); $("#editCommentReply").click(() => tryCatch(editCommentReply)); $("#deleteComment").click(() => tryCatch(deleteComment)); $("#deleteCommentReply").click(() => tryCatch(deleteCommentReply));
```

Comment:
Nit: Doesn't seem to be a line break at the semi-colons in the button handler assignments.